to-json-schema.js

```diff
@@ -145,6 +155,8 @@ async function convertAvroToJsonSchema(avroDefinition, isTopLevel) {
     return jsonSchema;
   }
 
+  validateAvroSchema(avroDefinition);
```

Comment:
the `validateAvroSchema` can be placed before the `if (isUnion)` test ? (before ligne 143)

Reply:
yeah, I think that would work too

Reply:
this can avoid executing the previous instructions :)